docs/one-click-integrations/github-integration.md

```diff
@@ -74,14 +82,31 @@ The following is a detailed description of how the Pulse GitHub integration auto
 
 -   Pulse associates all GitHub teams of the person who creates a Git tag with the corresponding deployment, excluding teams with less than two members. Pulse only takes changes to GitHub teams into account on Git tags created after those changes.
 
-### Use the CLI or API
+### Use the CLI or API (don't detect deployments automatically)
 
 -   Pulse doesn't detect deployments automatically using GitHub events.
 
     This is useful if none of the automatic deployment detection strategies match your workflow and you must have control over the way Pulse tracks your deployments.
 
 -   In this case, you must send to Pulse the information about your **deployments** and the corresponding **changes** using the [Pulse CLI](../cli/cli.md) or the [Ingestion API](https://ingestion.pulse.codacy.com/v1/api-docs).
 
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The following is a detailed description of how the Pulse GitHub integration automatically detects incidents:
+
+### Use pull request reverts (based on default branch)
+
+-   Pulse bases incident detection on [pull request reverts](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/reverting-a-pull-request).
```

Comment:
@afsmeira I added this link to GitHub documentation, but it's not as detailed as Bitbucket. For example, it doesn't refer to the created branch when a pull request is reverted.

Reply:
When reverting a PR, GitHub will not allow you to customize the branch name. I guess that's why they don't mention it.